### PR TITLE
Adds support for configurable user-agent that ignores logging

### PR DESCRIPTION
**Pull request by ImDevinC**
This change updates the ignored user-agent for logging to be a configurable value rather than harcoded

### CODE REVIEW

File: version [hunk 1]
```diff
@@ -1 +1 @@
-v1.4.10
\ No newline at end of file
+v1.4.11
```
Comment (echoboomer) [inline version:1]: Version should also be changed on line 11 in `backend/config.py`.

**Review by echoboomer — CHANGES_REQUESTED** [review]

**Comment by ImDevinC** [comment]
I'm currently traveling but will check tomorrow.

**Comment by ImDevinC** [comment]
I'm able to reproduce, and I know why. I will try to have a fix soon

**Comment by ImDevinC** [comment]
Addressed in https://github.com/echoboomer/incident-bot/pull/280

**Pull request merged**